lib/core/viewmodels/emergency_viewmodel.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:notredame/core/viewmodels/security_viewmodel.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class EmergencyViewModel extends SecurityViewModel {
```

Comment:
Because it's a single function to add, you can avoid this class and pass the `SecurityViewModel` directly to the `EmergencyView`

Reply:
I leaved it because there is now more than one function in this viewModel